common/OpTestUtil.py

```diff
@@ -1082,6 +1082,23 @@ def ping_fail_check(self, i_ip):
             return False
         return True
 
+    def check_pci_type(self, pci_device, cv_HOST):
+        print("SACHIN_DEBUG4 : from utils", pci_device)
+        self.bus_id = pci_device[0:7]
+        search_string = "OF_TYPE=network"
+        print("SACHIN_PB_DEBUG4:bus_id is", self.bus_id)
+        self.cmd = "cat /sys/class/pci_bus/{}/device/{}/uevent".format(self.bus_id, pci_device)
+        self.cmd_output = cv_HOST.host_run_command(self.cmd)
+        print("SACHIN_DEBUG4: cmd is ", self.cmd_output)
+        var = {}
+        for line in self.cmd_output:
+            if "=" in line:
+                name, value = line.split("=")
+                if name == "OF_TYPE":
+                    var[name] = str(value).rstrip()
+                    print("SACHIN_DEBUG4:Type of device is ", var[name])
+                    log.info("Check_pci_type:Type of PCI device is %s",  var[name])
```

Comment:
are we need to return value which can be used in test

Reply:
Done